package.json

```diff
@@ -23,7 +23,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-app-rewired start",
+    "start": "PORT=3006 react-app-rewired start",
```

Comment:
this syntax does not work on Windows. There should be other ways to define the port

Reply:
fixed